<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerfProfiler::Internal::PerfTracePointDialog</class>
 <widget class="QDialog" name="PerfProfiler::Internal::PerfTracePointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creating Memory Trace Points</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Run the following script as root to create trace points?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="privilegesLayout">
     <item>
      <widget class="QLabel" name="privilegeslabel">
       <property name="text">
        <string>Elevate privileges using:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="privilegesChooser">
       <item>
        <property name="text">
         <string notr="true">n.a.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">pkexec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">sudo</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PerfProfiler::Internal::PerfTracePointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PerfProfiler::Internal::PerfTracePointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
